modules/apps/27-interchain-accounts/controller/types/params.go

```diff
@@ -0,0 +1,59 @@
+package types
+
+import (
+	"fmt"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+const (
+	// DefaultControllerEnabled is the default value for the controller param (set to true)
+	DefaultControllerEnabled = true
+)
+
+var (
+	// KeyControllerEnabled is store's key for ControllerEnabled Params
+	KeyControllerEnabled = []byte("ControllerEnabled")
+)
+
+// ParamKeyTable type declaration for parameters
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
```

Comment:
So there's no registration issues here between controller/host as `NewKeyTable()` is creating separate instances, correct?

Reply:
Correct! The param types are also not equal

Reply:
Ah yes! I noticed after I wrote this comment the separation of host/controller subpackages in the protos :)